examples/any-type/any_type.description

```diff
@@ -1,4 +1,4 @@
-// The `any` data type is the root of the Ballerina data types. It can represent a variable of any data type.
+// The `any` data type is the root of the Ballerina data types. It can represent a variable of any type excluding error type.
```

Comment:
Need to rephrase this as well. It's misleading to say `any` is the root of data types. Shall we rephrase this to something like "The `any` type is the root of all the types except `error` in Ballerina." ?

Reply:
Kept this for till the definition gets updated in spec